doc/usage/extensions/math.rst

```diff
@@ -128,10 +129,12 @@ built:
 
 This extension puts math as-is into the HTML files.  The JavaScript package
 MathJax_ is then loaded and transforms the LaTeX markup to readable math live in
-the browser.
+the browser.  Reminder: you should use the math :rst:dir:`directives <math>` and
```

Comment:
Could you use `.. attension::` directive here?

Reply:
Done, good idea.